lib/selector/tokenizer/tokenizer.go

```diff
@@ -54,7 +54,7 @@ type Token struct {
 }
 
 const (
-	identifierExpr = `[a-zA-Z_./-][a-zA-Z0-9_./-]*`
+	identifierExpr = `[a-zA-Z0-9]?([a-zA-Z0-9_.-]{0,61}[a-zA-Z0-9])*`
```

Comment:
I think this should match the labelRegex no?  (i.e. the full thing)  We don't, unless I'm missing something, strip the subdomain prefix... or does the subdomain prefix never get passed to us?

@caseydavenport what say you?

Also, I think we need to make sure this regex is UT'd  -  I believe there are plenty of UTs in the selector codebase